neqo-transport/src/crypto.rs

```diff
@@ -821,6 +824,10 @@ pub struct CryptoStates {
 }
 
 impl CryptoStates {
+    fn initials_is_empty(&self) -> bool {
+        self.initials.iter().all(|(_, v)| v.is_none())
```

Comment:
You can use `Iterator::flatten` and `Iterator::count` with `EnumMap::values` for this.  I doubt that the code it generates would be more efficient, but it might be cheaper.